packages/browser-wallet/src/popup/styles/config/_functions.scss

```diff
@@ -0,0 +1,9 @@
+@use 'sass:math';
+
+@function strip-unit($value) {
+    @return math.div($value, ($value * 0 + 1));
```

Comment:
what is x * 0? :thinking: 
I'm confused by this

Reply:
It's a commonly known hackish solution to strip any number value from it's units, f.x. `strip-unit(18px) -> 18`.